### ISSUE TITLE
I want to use the  encoder part only as a CNN, what layers should I use and after which one I put the softmax layer?

### ISSUE THREAD

**Issue by fernande2000** [yzwxx/vae-celebA#5]
If I want to use the encoder only as a cnn, what layer is the one containing the embedded coding that I need to use before the soft max? that is, I want to use the encoder with the trained parameters as a pretrained CNN, But I am not sure what will be the layer that has to be connected to the softmax. This one is the equivalent the last layer of a CNN before the softmax layer, correct?

**Comment by yzwxx** [comment]
Yes.
